###### Class test.Test5 (test.Test5)
.class public Ltest/Test5;
.super Ljava/lang/Object;
.source "Test5.java"


# instance fields
.field flag:Z


# direct methods
.method public constructor <init>()V
    .registers 1

    .line 6
    invoke-direct {p0}, Ljava/lang/Object;-><init>()V

    return-void
.end method


# virtual methods
.method public test1_普通trycatch()V
    .registers 4

    .line 10
    :try_start_0
    sget-object v0, Ljava/lang/System;->out:Ljava/io/PrintStream;

    const-string v1, "try"

    invoke-virtual {v0, v1}, Ljava/io/PrintStream;->println(Ljava/lang/String;)V
    :try_end_7
    .catch Ljava/lang/RuntimeException; {:try_start_0 .. :try_end_7} :catch_8

    .line 13
    goto :goto_10

    .line 11
    :catch_8
    move-exception v0

    .line 12
    .local v0, "e":Ljava/lang/RuntimeException;
    sget-object v1, Ljava/lang/System;->out:Ljava/io/PrintStream;

    const-string v2, "catch"

    invoke-virtual {v1, v2}, Ljava/io/PrintStream;->println(Ljava/lang/String;)V

    .line 14
    .end local v0    # "e":Ljava/lang/RuntimeException;
    :goto_10
    return-void
.end method
